pdoc/doc.py

```diff
@@ -314,12 +314,19 @@ def members(self) -> dict[str, Doc]:
                     taken_from=taken_from,
                 )
             else:
+                try:
+                    default_value = self._member_objects["__pydantic_fields__"][
+                        name
+                    ].default
+                except KeyError:
+                    default_value = obj
```

Comment:
Let's move this logic into `pdoc/_pydantic.py` and shorten it here to something like this:

```suggestion
                default_value = _pydantic.default_value(self, name, obj)
```

This allows us to keep all pydantic-specific logic in a central location if we ever decide to break it out.

Reply:
SGTM